File: w06/d03/Classwork/makes10.rb
def makes10(a,b)
	if a + b == 10
		puts "true"
	elsif (a + b) - 10 == a || (a + b) - 10 == b
		puts "true"
	else
		puts "false"
	end
end

makes10(9, 10)

makes10(9, 9)

makes10(1, 9)

File: w06/d03/Classwork/makes10.js
var args = process.argv

var makes10 = function(num1, num2){

	if (num1 == 10) {
		console.log(true);
	} else if (num2 == 10){
		console.log(true);
	} else if (num1 + num2 == 10){
		console.log(true);
	} else{
		console.log(false);
	};

};
debugger //node debug filename.js --- cont --- repl

makes10(parseInt(args[2]), parseInt(args[3]));